### ISSUE TITLE
Dates are wrong

### ISSUE THREAD

**Issue by hewimeddel** [Manuel-777/MTG-Arena-Tool#344]
I am using Windows 10 in german language and als MTGA in german language.
The dates displayed in the "ECONOMY"-Tab are really wrong.

This has been the case in previous versions, then it was okay, and now it's wrong again since a few weeks and it is worse than it ever was before. Before it was only false dates, but the correct order, but now everything is wrong.

Maybe it has to do with the german formatting of date and time? 
It is   DD.MM.YYYY  HH:MM:SS  and this format is also used in the logfiles that MTGA creates.  

It is used inside the logs in C:\Users\{your Windows user}\AppData\LocalLow\Wizards Of The Coast\MTGA\output_log.txt and also for the naming of the logs in "C:\Program Files (x86)\Wizards of the Coast\MTGA\MTGA_Data\Logs\Logs and inside the logs there.

**Comment by Manuel-777** [comment]
Indeed, its a problem with having arena in other language other than English, unfortunately, date times are hard to parse when you have so many different plausible formats and styles. MTG Arena should *not* change the format of the logs..
Only solution at the moment and until either we come up with a solution or Arena fixes it is to use English language alone.

**Comment by Manuel-777** [comment]
Duplicate of #399 , will continue discussion there
